docs/system-tools/web-ui.md

```diff
@@ -4,4 +4,176 @@ title: Temporal Web UI
+
+## Deploying and Securing Temporal Web
+
+_This section is still being written - if you have specific questions you'd like us to answer, please search or [ask on GitHub](https://github.com/temporalio/web/)._
+
+### Configuring Temporal Web
+
+Set these environment variables if you need to change their defaults:
+
+| Variable                         | Description                                                                     | Default                       |
+| -------------------------------- | ------------------------------------------------------------------------------- | ----------------------------- |
+| TEMPORAL_GRPC_ENDPOINT           | String representing server gRPC endpoint                                        | 127.0.0.1:7233                |
+| TEMPORAL_WEB_PORT                | HTTP port to serve on                                                           | 8088                          |
+| TEMPORAL_CONFIG_PATH             | Path to config file, see [configurations](#configuring-authentication-optional) | ./server/config.yml           |
+| TEMPORAL_PERMIT_WRITE_API        | Boolean to permit write API methods such as Terminating Workflows               | true                          |
+| TEMPORAL_WEB_ROOT_PATH           | The root path to serve the app under                                            | /                             |
+| TEMPORAL_HOT_RELOAD_PORT         | HTTP port used by hot reloading in development                                  | 8081                          |
+| TEMPORAL_HOT_RELOAD_TEST_PORT    | HTTP port used by hot reloading in tests                                        | 8082                          |
+| TEMPORAL_SESSION_SECRET          | Secret used to hash the session with HMAC                                       | "ensure secret in production" |
+| TEMPORAL_EXTERNAL_SCRIPTS        | Additional JavaScript tags to serve in the UI                                   |                               |
+| TEMPORAL_GRPC_MAX_MESSAGE_LENGTH | gRPC max message length (bytes)                                                 | 4194304 (4mb)                 |
+
+<details>
+<summary>
+Optional TLS configuration variables:
+</summary>
+
+| Variable                              | Description                                                         | Default |
+| ------------------------------------- | ------------------------------------------------------------------- | ------- |
+| TEMPORAL_TLS_CERT_PATH                | Certificate for the server to validate the client (web) identity    |         |
+| TEMPORAL_TLS_KEY_PATH                 | Private key for secure communication with the server                |         |
+| TEMPORAL_TLS_CA_PATH                  | Certificate authority (CA) certificate for the validation of server |         |
+| TEMPORAL_TLS_ENABLE_HOST_VERIFICATION | Enables verification of the server certificate                      | true    |
```

Comment:
https://github.com/temporalio/temporal/releases/tag/v1.9.0

this flag was changed :)

Reply:
tls_disable_host_verification (default false) ;)